projects/js-packages/publicize-components/src/components/form/media-validation-notices.tsx

```diff
@@ -0,0 +1,41 @@
+import useAttachedMedia from '../../hooks/use-attached-media';
+import useFeaturedImage from '../../hooks/use-featured-image';
+import useImageGeneratorConfig from '../../hooks/use-image-generator-config';
+import useMediaDetails from '../../hooks/use-media-details';
+import useMediaRestrictions from '../../hooks/use-media-restrictions';
+import { NO_MEDIA_ERROR } from '../../hooks/use-media-restrictions/constants';
+import useSocialMediaConnections from '../../hooks/use-social-media-connections';
+import { InstagramNoMediaNotice } from './instagram-no-media-notice';
+import { MediaRequirementsNotice } from './media-requirements-notice';
+
+export const MediaValidationNotices: React.FC = () => {
+	const { connections } = useSocialMediaConnections();
+	const { isEnabled: isSocialImageGeneratorEnabledForPost } = useImageGeneratorConfig();
```

Comment:
We do not need to pass this down since we removed the Share as a Social post toggle; turning on SIG should not automatically result in the validation pass. Technically it could also be removed from the validation hook if I am right, but for now we can just not pass the prop down and that should do it

Reply:
Yes, you are right. We can remove it.